src/models/user-played-time-response.model.ts

```diff
@@ -0,0 +1,44 @@
+import type { GameConcept } from "./title-concept.model";
+import type { Media } from "./game-media.model";
+
+export interface GetUserPlayedTimeResponse {
+  /** The title ID of this game 
+      @example CUSA26431_00
+      */
+  titleId: string;
+
+  /** Default name for the title */
+  name: string;
+
+  /** Localized name for the title */
+  localizedName: string;
+
+  /** URL to game's console launch screen, from image.api.playstation.com */
+  imageUrl: string;
+
+  /** URL to localized game's console launch screen. Will default to imageUrl if not unique */
+  localizedImageUrl: string;
+
+  /** Type of game being played. Can be ps5_native_game, ps4_game or none */
+  category: string;
+
+  /** Service where this game was acquired from: EA Access, none(purchased/none_purchased, ps_plus, other, */
+  service: string;
+
+  /** Metadata about this game, including titleIds and localization for other regions, genres, name and id */
+  concept: GameConcept;
+
+  /** An object with audio, video, images properties that will contain arrays of objects with URL, format and type properties */
+  media: Media;
+
+  /** A Date object in ISO format */
+  firstPlayedDateTime: Date;
+
+  /** A Date object in ISO format */
+  lastPlayedDateTime: Date;
```

Comment:
I am confused by these. Are these ISO strings or actual date objects? We can strongly type both but native `Date` objects are their own thing.

Reply:
They should be ISO Strings because in my implementation I convert them to actual `Date` objects and then get their time in seconds with `.getTime()`.

For reference:

```
//? Instantiate time played data
const firstPlayed = Math.floor(
    new Date(title.firstPlayedDateTime ?? 0).getTime() / 1000
);
const lastPlayed = Math.floor(
    new Date(title.lastPlayedDateTime ?? 0).getTime() / 1000
);
```

Reply:
Sorry, I am unclear, can you explain this another way?

Reply:
Guess I'm overcomplicating it. It's an ISO string like the last played data that you get when fetching the user games. Same data type.